src/style.css

```
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+#header,
+#footer,
+.important,
+.interesting,
+.surprising {
+  height: 100px;
+  width: 300px;
+  margin-left: auto;
+  margin-right: auto;
```

better to write these rules in one line like this:
```suggestion
  margin: 0 auto;
```